docker-images/build-images.sh

```diff
@@ -7,6 +7,12 @@ MODE=${1:-"push"}
 DOCKER_TAG=${DOCKER_TAG:-"latest"}
 MVN_ARGS=${MVN_ARGS:-""}
 
+echo "Installing root pom"
+make java_install_single
+
+echo "Building Tracing dependency"
+make build --directory=tracing/
```

Comment:
Check whole `build-images.sh` ... as it was done before, it only picked-up `kafka` and `http` modules, since they only had lib dependencies.
Now there is `tracing` module dependency, which needs to be build before.
This was one way of achieving this ... I'm fine with any other way.